pkg/cli/admin/catalog/mirror.go

```diff
@@ -173,6 +174,7 @@ func NewMirrorCatalog(f kcmdutil.Factory, streams genericclioptions.IOStreams) *
 	flags.IntVar(&o.MaxPathComponents, "max-components", 2, "The maximum number of path components allowed in a destination mapping. Example: `quay.io/org/repo` has two path components.")
 	flags.StringVar(&o.IcspScope, "icsp-scope", o.IcspScope, "Scope of registry mirrors in imagecontentsourcepolicy file. Allowed values: repository, registry. Defaults to: repository")
 	flags.IntVar(&o.MaxICSPSize, "max-icsp-size", maxICSPSize, "The maximum number of bytes for the generated ICSP yaml(s). Defaults to 250000")
+	flags.BoolVar(&o.ContinueOnError, "continue-on-error", o.ContinueOnError, "If an error occurs while mirroring, keep going and attempt to mirror as much as possible.")
```

Comment:
should we explicitly call the default out as true here? otherwise we're losing that context when someone else comes along later

Reply:
agree, this changes the default behavior from true to false